### DISCUSSION TITLE
Error - terminated by signal SIGKILL (Forced quit)

### DISCUSSION THREAD

**Discussion by m0ham3dx** [community/community#29281] [Codespaces] [answered]
![sigterm](https://user-images.githubusercontent.com/82266083/184387610-2a26741a-73ac-4f0d-855c-6d55dd90ec29.gif)

Sometimes when executing a command which takes time to execute , i get a 
`terminated by signal SIGKILL(Forced Quit)`  - Issue , this in github codespaces. It erratic, sometimes it work and other times it doesnt. Any ideas on how to solve this ? issue is reproduced in the gif above

**Comment by legomushroom** [comment, marked as answer]
@m0ham3dx what is this test about? I know we have an OOM killer up but that reacts on large amount of RAM taken by a process to keep the connection to the Codespace up. Is it possible that this process is particularly RAM heavy?

**Reply by m0ham3dx** [reply]
> @m0ham3dx what is this test about? I know we have an OOM killer up but that reacts on large number of RAM taken by a process to keep the connection to the Codespace up. Is it possible that this process is particularly RAM heavy?

Thanks for the answer, your right , it was a ram issue , when I increased the size of the ram for the code space it worked 
![SigtermFix](https://user-images.githubusercontent.com/82266083/184411656-1a766d5b-8803-4d26-bc33-65f11dd386d6.gif)

**Reply by m0ham3dx** [reply]
> @m0ham3dx what is this test about? I know we have an OOM killer up but that reacts on large number of RAM taken by a process to keep the connection to the Codespace up. Is it possible that this process is particularly RAM heavy?

The test is compiling a smart contract which has buggy code , to simulate a bad coding patterns, the test is an online compiler that is compiling the smart contract and then querying a json rpc , to find the corresponding block in the blockhain where the bug was present.
